frontend/VolunteerOne/screens/Friends/index.js

```diff
@@ -1,44 +1,56 @@
 import React from 'react';
 import { StyleSheet, Dimensions, ScrollView } from 'react-native';
 import { Block, theme } from 'galio-framework';
+import { Button } from "../../components";
+
+// import { Card } from '../../components';
+import NotificationItem from '../../components/NotificationItem';
+// import articles from '../../constants/articles';
+import notifications from '../../constants/notifications';
+
+// <matt>
+// import { Block, Button, Text, theme } from "galio-framework";
+// </matt>
 
-import { Card } from '../../components';
-import articles from '../../constants/articles';
 const { width } = Dimensions.get('screen');
 
 class Home extends React.Component {
-  renderArticles = () => {
+  renderNotifications = () => {
     return (
       <ScrollView
         showsVerticalScrollIndicator={false}
-        contentContainerStyle={styles.articles}>
+        contentContainerStyle={styles.notifications}>
+ 
         <Block flex>
-          <Card item={articles[0]} horizontal  />
-          <Block flex row>
-            <Card item={articles[1]} style={{ marginRight: theme.SIZES.BASE }} />
-            <Card item={articles[2]} />
-          </Block>
-          <Card item={articles[3]} horizontal />
-          <Card item={articles[4]} full />
+          <NotificationItem item={notifications[0]} horizontal  />
```

Comment:
It's fine leaving it here for now, but we will need to render the <NotificationItem> in the Header component when you click the Bell Icon

Reply:
I added a comment to let others know that the page is being used as a temporary test page